sdk/pim-eligibility-migration/eligibility_sync.py

```
@@ -0,0 +1,524 @@
+
+
+def _str_list_change_summary(ov: list, nv: list) -> str:
+    """Compact summary for two string-id lists (order ignored for equality; delta for logging)."""
+    bs = set(map(str, ov))
+    afs = set(map(str, nv))
+    removed = len(bs - afs)
+    added = len(afs - bs)
+    n_o, n_n = len(ov), len(nv)
+    if removed == 0 and added == 0:
+        return f"{n_o} ids"
+    return f"{n_o}→{n_n} ids (−{removed}/+{added})"
```

Ruff (0.15.9)

[warning] String contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)